checkov/bicep/runner.py

```diff
@@ -68,7 +68,7 @@ def __init__(
 
         self.context: dict[str, dict[str, Any]] = {}
         self.definitions: dict[Path, BicepJson] = {}  # type:ignore[assignment]  # need to check, how to support subclass differences
-        self.definitions_raw: dict[Path, list[tuple[int, str]]] = {}
+        self.definitions_raw: dict[Path, list[tuple[int, str]]] = {}    # type:ignore[assignment]
```

Comment:
@maxamel Why is this ignore needed?

Reply:
Because the parent class now has this field as well, and it's defined differently in each child class. 
I think it's the same case as the above line with `self.definitions`, and it's ignored there for the same reason.